tapcfg/server.go

```diff
@@ -624,6 +625,32 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 		},
 	)
 
+	// Set up the supply verifier, which validates supply commitment leaves
+	// published by asset issuers.
+	//
+	// Initialize the database backend for the supply syncer.
+	supplySyncerDb := tapdb.NewTransactionExecutor(
+		db, func(tx *sql.Tx) tapdb.BaseUniverseStore {
+			return db.WithTx(tx)
+		},
+	)
+	supplySyncerStore := tapdb.NewSupplySyncerStore(supplySyncerDb)
```

Comment:
This can just be: `tapdb.NewSupplySyncerStore(uniDB)` as that has the same executor signature.